west.yml

```diff
@@ -74,6 +74,9 @@ manifest:
     - name: hal_stm32
       revision: f8ff8d25aa0a9e65948040c7b47ec67f3fa300df
       path: modules/hal/stm32
+    - name: hal_telink
+      revision:
```

Comment:
Can you please point this to a revision in a `hal_telink` repo hosted by yourself so we can take a look before creating the repo? You can use the `url` tag to point to a repo outside `zephyrproject-rtos`

Reply:
Done